nvflare/app_common/np/np_model_locator.py

```diff
@@ -29,18 +29,21 @@
 class NPModelLocator(ModelLocator):
     SERVER_MODEL_NAME = "server"
 
-    def __init__(self, model_dir="models", model_name="server.npy"):
+    def __init__(self, model_dir="models", model_names=None):
```

Comment:
You can make it backward compatible by allowing "model_name" to be either a str or a dict.

Reply:
Made it backward compatible.

Reply:
Checked the file, but "model_names" still there. To be backward compatible, I thought we would keep the arg name to be "model_name" (instead of model_names), and allow "model_name" to be either str or dict.

Since NPModelLocator has been existing and some customers may have been using it. If we change the arg name, then those customer's code would be broken.  Actually some unit tests are already broken since they use the arg name "model_name".